Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -495,6 +497,10 @@ class LocalDOMWindow final
     // been activated, while negative infinity indicates that a user activation-gated API has consumed the last user activation of W. The initial
     // value is positive infinity.
     MonotonicTime m_lastActivationTimestamp { MonotonicTime::infinity() };
+    // For the purpose of tracking user activation, each Window W has a last history-action activation timestamp. This is a number indicating the last time W got
+    // an activation notification. It corresponds to a DOMHighResTimeStamp value except for one case: positive infinity indicates that W has never
+    // been activated. The initial value is positive infinity.
+    MonotonicTime m_lastHistoryActionActivationTimestamp { MonotonicTime::infinity() };
```

Comment:
This comment is a bit verbose.  The initial value is clear from the code.

Reply:
Fair enough will remove as part of any required changes

Reply:
Removed